bin/riak-bucket-exporter.js

```diff
@@ -19,6 +22,9 @@ var bucket = program.args;
 program.host = program.host || 'localhost';
 program.port = program.port || '8098';
 program.file = program.file || bucket+'.json';
+program.concurrency = program.concurrency || 20;
+program.meta = program.meta || false;
```

Comment:
guess this will come in as a string. so setting --meta false would still make it true as "false" is interpreted as true. perhaps we should do `program.meta = (program.meta==='true') || false;` that one should to the trick